bindings/python/unicorn/unicorn.py

```diff
@@ -223,6 +223,9 @@ class uc_x86_xmm(ctypes.Structure):
         ("high_qword", ctypes.c_uint64),
     ]
 
+class uc_arm64_neon128(uc_x86_xmm):
```

Comment:
Not sure if duplicating code is something we want. Maybe we could have a mother class like `vector128` and then 2 child classes `uc_x86_xmm` and `uc_arm64_neon128` inherit from it?